src/input.c

```diff
@@ -1088,13 +1088,12 @@ void process_mouse_over(int x, int y)
 				if (subWindow == NULL)
 					break;
 
-				ebx = ebx & 0xFFFFFF00;
 				edi = cursorId;
 				esi = (int)subWindow;
 				// Not sure what this is for, no windows actually implement a handler
 				// RCT2_CALLFUNC_X(subWindow->event_handlers[WE_UNKNOWN_0E], &eax, &ebx, &ecx, &edx, &esi, &edi, &ebp);
 				cursorId = edi;
-				if ((ebx & 0xFF) != 0)
+				if (0)
```

Comment:
Well, if you don't want this removed, we can work around that. The warning here is actually at line 1091 about `ebx` being uninitialized. It's the first time `ebx` is being assigned, and it's being read before the write. I'd assume if that line is there, there's supposed to have been a value assigned to it before that. Was there something before that should be setting the value to the variable, or should I just go ahead and change that to `ebx = 0;`?